project-base/app/src/Migrations/Version20240722095556.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20240722095556 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE stores RENAME COLUMN locationLatitude to latitude');
+        $this->sql('ALTER TABLE stores RENAME COLUMN locationLongitude to longitude');
+    }
+
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function down(Schema $schema): void
```

Comment:
you can remove this method completely, we do not maintain the backward migrations

Reply:
Ok I have removed the content but left the method there (followed the example from other migrations).